### PR TITLE
global: adapt to october release

### PR DESCRIPTION
**Pull request by ppanero**

### CODE REVIEW

File: docs/develop/run.md [hunk 1]
```diff
@@ -21,15 +21,14 @@ Are we done? Yes, let the fun begin...
 
 ### List records
 
-Let's see what is in the instance by querying the API. Note that localhost and 127.0.0.1 can
-be used interchangeably. Using another terminal:
+!!! warning "Use 127.0.0.1"
+    Due to CSP it is important that you use 127.0.0.1, and not localhost. Unless you ser the `SERVER_HOSTNAME` to localhost.
```
Comment (zzacharo) [inline docs/develop/run.md:25]: typo `ser` -> `set` :)

File: docs/preview/index.md [hunk 2]
```diff
@@ -142,6 +142,9 @@ Visit https://127.0.0.1
 firefox https://127.0.0.1
 ```
 
+!!! warning "Use 127.0.0.1"
+    Due to CSP it is important that you use 127.0.0.1, and not localhost. Unless you ser the `SERVER_HOSTNAME` to localhost.
```
Comment (zzacharo) [inline docs/preview/index.md:146]: Same here :)

File: docs/develop/run.md (hunk 1 above)
Comment (lnielsen) [inline docs/develop/run.md:25]:
```suggestion
    Due to CSP it is important that you use 127.0.0.1, and not localhost. Unless you set the `SERVER_HOSTNAME` to localhost.
```

File: docs/preview/index.md (hunk 2 above)
Comment (lnielsen) [inline docs/preview/index.md:146]:
```suggestion
    Due to CSP it is important that you use 127.0.0.1, and not localhost. Unless you set the `SERVER_HOSTNAME` to localhost.
```

File: docs/develop/make_it_your_own.md [hunk 3]
```diff
@@ -95,9 +95,30 @@ Then, run the `invenio-cli update --no-install-js` command as above and refresh
...
+return (
+    <Item key={index} href={`/records/${record.id}`}>
+    <Item.Content>
+        <Item.Header>{record.metadata.titles[0].title}</Item.Header>
+        <Item.Description>
+        {_truncate(record.metadata.descriptions[0].description, { length: 400 })}
+        </Item.Description>
+    </Item.Content>
+    </Item>
+)
```
Comment (fenekku) [inline docs/develop/make_it_your_own.md:117]: One level of indentation.

File: docs/develop/make_it_your_own.md [hunk 4]
```diff
@@ -286,6 +310,9 @@ You are now a master of the metadata model!
 
 ## Change permissions
 
+!!! error "Temporarily not supported"
+    This functionality is temporarily disabled.
+
```
Comment (fenekku) [inline docs/develop/make_it_your_own.md:315]: Ah, good to know. I will add it to the ticket about configuration I making.

File: docs/develop/run.md [hunk 5]
```diff
@@ -21,15 +21,14 @@ Are we done? Yes, let the fun begin...
 
 ### List records
 
-Let's see what is in the instance by querying the API. Note that localhost and 127.0.0.1 can
-be used interchangeably. Using another terminal:
+!!! warning "Use 127.0.0.1"
+    Due to CSP it is important that you use 127.0.0.1, and not localhost. Unless you set the `SERVER_HOSTNAME` to localhost.
```
Comment (fenekku) [inline docs/develop/run.md:25]: :100:

**Review by fenekku — COMMENTED** [review]
Thanks for the update. There is a minor issue with the one level of indentation missing in the search example. Minor though :)

**Pull request merged**